Unify schema into modeling rules (#2165)
demisto/demisto-sdk: CHANGELOG.md

* Added a **validate** check, making sure classifier id and name values match. Updated the classifier **format** to update the id accordingly.

> * The **generate-docs** command will now auto-generate the playbook image link as default. Added the `--custom-image-link' argument to override.

Reviewer: There's a bug here, can you fix it also?
```suggestion
* The **generate-docs** command will now auto-generate the playbook image link by default. 
* Added the `--custom-image-link` argument to override.
```

* Removed a traceback line from the **init** command templates: BaseIntegration, BaseScript.
* Updated the token in **_add_pr_comment** method from the content-bot token to the xsoar-bot token.
* Added a new validation to the **validate** command to verify that the docker in use is not deprecated.

> * Added support in the **unify** command to unify the schema into the Modeling Rule.

Reviewer: ```suggestion
* Added support in the **unify** command to unify a schema into its Modeling Rule.
```
